benchmarks/python_e2e/main.py

```diff
@@ -130,6 +130,7 @@ def run(algos,
         success = benchmark.run()
 
         algo_name = benchmark.results[1].name
+        algo_name = f"benchmarks.{algo_name}"
```

Comment:
This seems like a questionable merge issue.

Reply:
Looks like it came from the auto-merge. Which one should I keep?

Reply:
@rlratzel ?

Reply:
I think the code was just written across two lines using the same var name, it could have been combined into a single statement. You should keep the code with both lines 132 and 133 as-is.

Reply:
I'm confused.  Doesn't this set `algo_name` to a value and then set `algo_name` to a different value?  What is the benefit of setting the same variable to two different values on successive lines?

Reply:
The second assignment to `algo_name` is using the value from first assignment.  I'm guessing it was done this way to make it easier to see the two steps that form the final name.  It could have been written in one line as: 
```
# prepend the string "benchmark." to the name attribute of the benchmark result obj
algo_name = f"benchmarks.{benchmark.results[1].name}"
```

Reply:
It does look like this is not part of your PR but something that was broken (if broken) somewhere else.  I would ignore this for now, we can correct it in a separate PR.